flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkClient.java

```diff
@@ -325,4 +334,27 @@ private ActorRef getJobManager() throws IOException {
 				actorSystem, AkkaUtils.getLookupTimeout(configuration));
 	}
 
+	@SuppressWarnings({ "unchecked", "rawtypes" })
+	static void addStormConfigToTopology(FlinkTopology topology, Map conf) throws ClassNotFoundException {
+		if(conf != null) {
```

Comment:
The majority of the Flink code uses spaces after keywords (if, for, ...). We have no strict checkstyle for that, but if we actually add a variant of Google code style, it helps when code is already in style.